src/Components/TodoappFilter/TodoappFilter.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cn from 'classnames';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  filterBy: Filter,
+  onFilterClick: (value: Filter) => void;
+};
+
+export const TodoappFilter: React.FC<Props> = ({
+  filterBy,
+  onFilterClick,
+}) => {
+  return (
+    // {/* // Active filter should have a 'selected' class */}
+    <nav className="filter" data-cy="Filter">
+      {Object.values(Filter).map(option => (
+        <a
+          key={option}
+          href={`#/${Filter.ALL ? '' : option.toLowerCase()}`}
```

Comment:
Don't really get why you need `#`

Reply:
Honestly... I don't know why. But it just was not working other way, and know when I've tried to remove it several tests failed